### PR TITLE
Fix #12123: Long server descriptions are not cut off properly

### LINKED ISSUE #12123: [UI] Long server_description should be cutted properly
**OS:** All
**Version:** 0.2.6
**Commit/Build:** 99b39f0

**Screenshots / Video:**
image

Long ``server_description`` may overflow the multiplay window, I think the text should be shortened by ellipsis properly :)

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
Before:
Screenshot 2020-07-04 at 17 58 14 copy

After:
Screenshot 2020-07-04 at 18 51 52

### CODE REVIEW

File: src/openrct2-ui/windows/ServerList.cpp [hunk 1]
```diff
@@ -463,17 +463,30 @@ static void window_server_list_scrollpaint(rct_window* w, rct_drawpixelinfo* dpi
 
         screenCoords.x = 3;
 
-        // Draw server information
-        if (highlighted && !serverDetails.Description.empty())
+        // Before we draw the server info, we need to know how much room we'll need for player info.
+        char players[32];
+        players[0] = 0;
```
Comment (AaronVanGeffen) [inline src/openrct2-ui/windows/ServerList.cpp:468]: I just moved this from its original position below, but we might as well refactor it into something like
```suggestion
        char players[32] = { 0 };
```

Comment (Gymnasiast) [reply src/openrct2-ui/windows/ServerList.cpp:468]: What's holding you back? It's a very minor change and unlikely to hinder code review.

Comment (AaronVanGeffen) [reply src/openrct2-ui/windows/ServerList.cpp:468]: Not being on my computer, for starters.

**Comment by Gymnasiast** [comment]
@tupaschoal Can you add one?

**Comment by tupaschoal** [comment]
Done on https://github.com/OpenRCT2/OpenRCT2/commit/c8e844d1003a6d358eb560762687e2c512dda373

**Pull request merged**